backend/game/cards/keep_cards/health_manipulation_cards/even_bigger.py

```diff
@@ -0,0 +1,20 @@
+from game.cards.card import Card
+from game.cards.keep_card import KeepCard
+
+
+class EvenBigger(KeepCard):
+    def __init__(self):
+        super(KeepCard, self).__init__("Even Bigger", 4,
+                                         "Your maximum [Heart] is increased by 2. Gain 2[Heart] when you get this card",
+                                         "")
+
+    def immediate_effect(self, player_that_bought_the_card, other_players):
+        KeepCard.immediate_effect(self, player_that_bought_the_card, other_players)
+        EvenBigger().special_effect(player_that_bought_the_card, None)
+
+    def discard_effect(self, player_that_bought_the_card, other_players):
+        KeepCard.discard_effect(self, player_that_bought_the_card, other_players)
+        player_that_bought_the_card.update_max_health_by(-2)
+
+    def special_effect(self, player_that_bought_the_card, other_players):
+        player_that_bought_the_card.update_max_health_by(2)
```

Comment:
I believe it should not just add to the max health but add to the regular health as well.

Reply:
see https://github.com/borisv13/kot/pull/73#discussion_r355237931